### PR TITLE
fix(assets): Adjust upri handling

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/13167

## What is the new behavior?

- Ensures that the upri handling is done after its inputs have been filled
- Marks upri files as not publishable to avoid https://github.com/xamarin/xamarin-macios/blob/8b3bf352d16d68b493c578be57ffe67bf37043c6/msbuild/Xamarin.MacDev.Tasks/Tasks/ComputeBundleLocationTaskBase.cs#L368

### <samp>🤖 Generated by Copilot at e37c2d7</samp>

This pull request optimizes the web assembly build and publish process by adjusting the content items and targets in the `uno.ui.tasks.assets.targets` file.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets [hunk 1]
```diff
@@ -198,7 +199,7 @@
 	as nuget packages content.
 	-->
 	<Target Name="_UnoAssetsGetCopyToPublishDirectory"
-	BeforeTargets="GetCopyToPublishDirectoryItems">
+			BeforeTargets="GetCopyToPublishDirectoryItems;_UnoGeneratePriMarker">
```
Comment (Youssef1313) [inline src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets:202]: I have already tried it but it didn't work at that time for some reason. https://github.com/unoplatform/uno/pull/13156 but it broke template tests on CI.

Comment (jeromelaban) [reply src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets:202]: Thanks, will validate further!

**Review by Youssef1313 — APPROVED** [review]
Nice find!

@kazo0 we should be able to remove the workaround once the new package is published.

**Comment by jeromelaban** [comment]
Actually, the workaround duplicates the assets in the nuget package, so it's likely required to remove it :)

**Comment by jeromelaban** [comment]
@mergifuio backport release/stable/4.10 legacy/4x

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/4.10 legacy/4x

**Pull request merged**